core/controllers/outgoing_emails_test.py

```diff
@@ -14,37 +14,41 @@
 
 """Tests for processing contact form submissions. """
 
+from core.tests import app_engine_test_base
 from core.controllers import outgoing_emails
 from core.utility import string_validator
-from core.tests import app_engine_test_base
+
 import config
 import main
 
 
+FORWARD_EMAIL_HANDLER = outgoing_emails.ForwardToAdminEmailHandler()
```

Comment:
Don't think you need the "()" at the end if it's a classmethod?

Reply:
You are absolutely correct. I kept thinking of that statement like I am initializing an object.

Removed!